cli/crates/cli/src/cli_input/publish.rs

```diff
@@ -36,6 +36,10 @@ pub struct PublishCommand {
     #[arg(long, default_value_t = 4000)]
     pub(crate) dev_api_port: u16,
 
+    /// The message to annotate the publication with
+    #[arg(long)]
```

Comment:
can we add the shortcut -m for --message?

Reply:
Sure!